README.md

```diff
@@ -187,6 +187,8 @@ Options are specified with the -D flag.  The allowed options include:
     PYTHON_EXECUTABLE=(/path/to/python_version): Path to specific Python executable
     BUILD_JULIA_BINDINGS=(ON/OFF): whether or not to build Julia bindings
     JULIA_EXECUTABLE=(/path/to/julia): Path to specific Julia executable
+    BUILD_GO_BINDINGS=(ON/OFF): whether or not to build Go bindings
+    GO_EXECUTABLE=(/path/to/go): Path to specific Go executable
```

Comment:
Hey should we also add `BUILD_GO_SHLIB` option here?
This option allows us to build shared libraries required by Go-bindings.

Reply:
Sure, Makes sense.

Reply:
I had use `BUILD_GO_SHLIB` because when we ask the user to download go-bindings, then we also ask the user to build these go-shared libraries on their own, and this option will help the user to just build the shared libraries required by the go-bindings and not the complete go-bindings (.go and .h files). 
I am not sure, if we should add this option or not.